packages/design-system-tokens/src/themes/medicare/defaultTheme.ts

```diff
@@ -248,6 +248,16 @@ const components = {
   'pagination-page-count__color': themeColors.gray,
   // review
   'review__border-color': themeColors['border'],
+  // steplist
+  steplist__color: themeColors.muted,
+  'steplist__color--current': themeColors.primary,
+  'steplist__background-color--current': themeColors.primary,
+  'steplist-step__color': themeColors.muted,
+  'steplist-step__border-color': themeColors.border,
+  'steplist-step__border-color--default': themeColors.muted,
```

Comment:
It took me a while to figure out that these border color variables had nothing to do with each other—that they're styling two different kinds of elements (the dividing line and the step number). I don't think there's anything you can do to reduce that confusion, but I'm just noting that. The `--state` naming convention does give you a clue because there are other variables with that postfix.

Reply:
Yeah in this case since all the \`--default\` variables go together it makes a little sense but isn't super understandable.. I think when I get to the a plugin for storybook possibly to theme these it won't matter as much. I was thinking last night as I drifted off to sleep about how creating a new theme would look, as it is now it's a little rough, lots of variables, lots of testing.. a simple editor of some type would improve that a lot.. On my mind :thinking: